person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  # accessor get method
+  attr_reader :id
+
+  # accessor get and set method
+  attr_accessor :name, :age
+
+  # private method
+  private
+
+  def of_age?
+    age >= 18
+  end
+
+  # publick method
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
- [ ] As you mentioned this method is public but now it's private because you put the `private` word above the two methods that make automatically the two methods private. So either you move this method above the `of_age` method or you make it public by adding the word `public` just above the `can_use_services` method 🙃 🙃